File: springboot-jdbc-data/src/main/java/com/codegik/poc/domain/Person.java
package com.codegik.poc.domain;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("person")
public class Person {
    @Id
    private Long id;
    private String name;

    public Person() {}
    public Person(Long id, String name) {
        this.id = id;
        this.name = name;
    }
    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public String getName() { return name; }
    public void setName(String name) { this.name = name; }
}

File: kotlin-validation/src/main/kotlin/com/codegik/poc/validation/domain/Person.kt
package com.codegik.poc.validation.domain

import com.codegik.poc.validation.annotation.Email
import com.codegik.poc.validation.annotation.MaxSize

data class Person(
    var name: String,
    @MaxSize(size = 100)
    var age: Int,
    @Email
    var email: String
)
